examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -8,7 +8,6 @@ import { treeDataObjectInternal } from "@fluid-experimental/tree-react-api/inter
 import { Inventory, treeConfiguration } from "./schema.js";
 
 export const InventoryListFactory = treeDataObjectInternal(
-	"tree",
```

Comment:
Since the tree is no longer being inserted into a directory, this `key` property is no longer needed.